README.rst

```diff
@@ -192,6 +192,12 @@ Usage
      # run_test_case = True (default value for 'run' parameter)
      # connection_error_strategy [strict|skip|ignore] Choose how to handle connection errors
 
+   You can set the password field by setting the PYTEST_JIRA_PASSWORD environment variable:
+
+    .. code:: sh
+
+      set PYTEST_JIRA_PASSWORD="FOO"
```

Comment:
didn't you mean `export` instead of `set` ?

Reply:
yeah, oops :)